start.py

```diff
@@ -102,5 +102,10 @@ def failure(timeout):
     primary_node.execute('sudo -u mapr maprcli stream create -path /sample-stream '
                          '-produceperm p -consumeperm p -topicperm p')
 
+    logger.info('Creating sdc user directory in MapR-FS ...')
+    create_sdc_user_directory_command = ['sudo -u mapr hadoop fs -mkdir -p /user/sdc',
+                                         'sudo -u mapr hadoop fs -chown sdc:sdc /user/sdc']
+    primary_node.execute("bash -c '{}'".format('; '.join(create_sdc_user_directory_command)))
```

Comment:
No need for `bash -c` anymore; automatically done since [this commit](https://github.com/clusterdock/clusterdock/commit/b5f48006cfd83494b43196b5b239235a34d38641).